scopes/defender/tester/ui/tests-page.tsx

```diff
@@ -80,7 +82,18 @@ export function TestsPage({ className }: TestsPageProps) {
   // TODO: change loading EmptyBox
   if (testData?.loading) return <TestLoader />;
 
+  const env = component.environment?.id;
+  const EmptyStateTemplate = emptyState.get(env || ''); // || defaultTemplate;
```

Comment:
maybe make `defaultTempalte = EmptyBox`?

Reply:
will add the mdx files here instead of emptyBox in the next pr